cub/cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -58,6 +58,10 @@
 
 #include <thrust/system/cuda/detail/core/triple_chevron_launch.h>
 
+#include <cuda/functional>
```

Comment:
Is that include still needed?

Reply:
I added it for `cuda::proclaim_return_type`. Should I include a different header?

Reply:
Offline discussion: it's the correct include.